frontend/add-client/form-inputs/intake-services-inputs.component.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+
+export default React.forwardRef<ServicesRef, IntakeServicesInputsProps>(
```

Comment:
Pulled this component out of services.component so we could reuse this part in the view-edit-intake-services.component